cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -80,9 +80,15 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
 
     prepareAndTestQuery(
       title = "List constraints",
-      text = "Calling the built-in procedure `db.constraints` will list all constraints, including their names.",
-      queryText = "CALL db.constraints",
+      text =
+        """Listing all constraints can be done with `SHOW CONSTRAINTS`, which will produce a table with these columns:
+          |include::list-constraints-table-columns.asciidoc[]""".stripMargin,
```

Comment:
Here the link works but it looks a bit weird, I don't think the `.Lists constraints output` should be there?

![Screenshot 2020-10-21 at 10 09 39](https://user-images.githubusercontent.com/7222590/96692080-f102c200-1385-11eb-8c75-9549490e3ef4.png)

Reply:
hmmm, in general that should have been the table descritpion/header

Reply:
so strange, no clue why this happens as it works fine for `list-users-table-columns.asciidoc` and `p("include::list-users-table-columns.asciidoc[]")`